src/Nethermind/Nethermind.Optimism.Test/OptimismPayloadAttributesTests.cs

```diff
@@ -44,15 +53,20 @@ public void Compute_PayloadID_with_EIP1559Params((string HexStringEIP1559Params,
         payloadId.Should().Be(testCase.PayloadId);
     }
 
-    private static IEnumerable<(byte[]?, PayloadAttributesValidationResult, PayloadAttributesValidationResult)> Validate_EIP1559Params_Holocene_TestCases()
+    private static IEnumerable<(byte[]?, PayloadAttributesValidationResult, PayloadAttributesValidationResult, PayloadAttributesValidationResult)> Validate_EIP1559Params_Holocene_Jovian_TestCases()
     {
-        yield return (null, PayloadAttributesValidationResult.Success, PayloadAttributesValidationResult.InvalidPayloadAttributes);
-        yield return (new byte[7], PayloadAttributesValidationResult.InvalidPayloadAttributes, PayloadAttributesValidationResult.InvalidPayloadAttributes);
-        yield return (new byte[8], PayloadAttributesValidationResult.InvalidPayloadAttributes, PayloadAttributesValidationResult.Success);
-        yield return (new byte[9], PayloadAttributesValidationResult.InvalidPayloadAttributes, PayloadAttributesValidationResult.InvalidPayloadAttributes);
+        yield return (null, PayloadAttributesValidationResult.Success, PayloadAttributesValidationResult.InvalidPayloadAttributes, PayloadAttributesValidationResult.InvalidPayloadAttributes);
+        yield return (new byte[7], PayloadAttributesValidationResult.InvalidPayloadAttributes, PayloadAttributesValidationResult.InvalidPayloadAttributes, PayloadAttributesValidationResult.InvalidPayloadAttributes);
+        yield return (new byte[8], PayloadAttributesValidationResult.InvalidPayloadAttributes, PayloadAttributesValidationResult.Success, PayloadAttributesValidationResult.InvalidPayloadAttributes);
+        yield return (new byte[9], PayloadAttributesValidationResult.InvalidPayloadAttributes, PayloadAttributesValidationResult.InvalidPayloadAttributes, PayloadAttributesValidationResult.InvalidPayloadAttributes);
+        yield return (new byte[15], PayloadAttributesValidationResult.InvalidPayloadAttributes, PayloadAttributesValidationResult.InvalidPayloadAttributes, PayloadAttributesValidationResult.InvalidPayloadAttributes);
+        yield return (new byte[16], PayloadAttributesValidationResult.InvalidPayloadAttributes, PayloadAttributesValidationResult.InvalidPayloadAttributes, PayloadAttributesValidationResult.Success);
+        yield return (new byte[17], PayloadAttributesValidationResult.InvalidPayloadAttributes, PayloadAttributesValidationResult.InvalidPayloadAttributes, PayloadAttributesValidationResult.InvalidPayloadAttributes);
     }
-    [TestCaseSource(nameof(Validate_EIP1559Params_Holocene_TestCases))]
-    public void Validate_EIP1559Params_Holocene((byte[]? Eip1559Params, PayloadAttributesValidationResult BeforeHolocene, PayloadAttributesValidationResult AfterHolocene) testCase)
+    [TestCaseSource(nameof(Validate_EIP1559Params_Holocene_Jovian_TestCases))]
+    public void Validate_EIP1559Params_Holocene_Jovian((byte[]? Eip1559Params, PayloadAttributesValidationResult BeforeHolocene,
```

Comment:
Same here, maybe it's better to have a separate test with "before/after Jovian" than mixing it with Holocene.

Note how for Holocene we had an implicit "after Granite" in "before Holocene": this case is similar, where we have an implicit "after Holocene" in "before Jovian"